### PR TITLE
Added ::fun goosed command

### PR DESCRIPTION
**Pull request by MrNesli**
#27 
Command added - ::fun goosed
goosed

### CODE REVIEW

**Review by PenguinBoi12 — CHANGES_REQUESTED** [review]
That's pretty good, well done. There's a few minor things that need to be adjusted. In general, I would've changed them myself because they are really minor but since it's your first PR I for Grace thought it would be better like that.

File: bot/extensions/fun_cog.py [hunk 1]
```diff
@@ -8,10 +8,14 @@
 from bot.models.extensions.fun.answer import Answer
 from discord.colour import Colour
 
-
```
[the remark this answers is not in the document]
Comment (MrNesli) [reply bot/extensions/fun_cog.py:11]: Oh

[the remark this answers is not in the document]
Comment (PenguinBoi12) [reply bot/extensions/fun_cog.py:11]: I forgot to send you the link, here it is https://peps.python.org/pep-0008/#blank-lines

**Review by PenguinBoi12 — APPROVED** [review]
Awesome, well done!

File: bot/extensions/fun_cog.py [hunk 2]
```diff
@@ -34,6 +38,16 @@ async def eightball_command(self, ctx, question):
 
         await ctx.send(embed=answer_embed)
 
+    @fun_group.command(name='goosed', help='Go goose yourself')
+    async def goose_command(self, ctx):
+        goosed_text = '**GET GOOSED**'
```
[the remark this answers is not in the document]
Comment (MrNesli) [reply bot/extensions/fun_cog.py]: Yeah, fair enough. I just thought for readability would be good to make it in another variable, but it doesn't really matter

**Pull request merged**